irssi/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fedora
+MAINTAINER http://fedoraproject.org/wiki/Cloud
+
+# Updating dependencies, installing irssi and cleaning dnf caches to reduce container size
+RUN dnf -y update && dnf -y install irssi irssi screen && dnf clean all
+
+# Starting a new Screen session called chat
+CMD screen -S chat
```

Comment:
what is this CMD entry for?  I thought that only the last CMD command was respected.